generators/entity-server/templates/src/main/java/package/web/rest/EntityResource.java.ejs

```diff
@@ -164,14 +179,19 @@ public class <%= entityClass %>Resource {
      * @param filter the filter of the request<% } %>
      * @return the ResponseEntity with status 200 (OK) and the list of <%= entityInstancePlural %> in body
      */
-    @GetMapping("/<%= entityApiUrl %>")<%- include('../../common/get_all_template', {asEntity, asDto, viaService}); -%>
+    @GetMapping("/<%= entityApiUrl %>")
+    <%_ if (databaseType === 'sql' && isUsingMapsId === true && !viaService) { _%>
+    @Transactional(readOnly = true) <%_ } _%>
```

Comment:
Sorry for old comment, but after this PR, there are some useless line, because it is not correctly formatted:

```java
/**
     * GET  /blablas : get all the blablas.
     *
     * @return the ResponseEntity with status 200 (OK) and the list of blablas in body
     */
    @GetMapping("/blablas")
    
    public List<Blabla> getAllBlablas() {
        log.debug("REST request to get all Blablas");
        return blablaService.findAll();
    }

    /**
     * GET  /blablas/:id : get the "id" blabla.
     *
     * @param id the id of the blabla to retrieve
     * @return the ResponseEntity with status 200 (OK) and with body the blabla, or with status 404 (Not Found)
     */
    @GetMapping("/blablas/{id}")

    public ResponseEntity<Blabla> getBlabla(@PathVariable Long id) {
        log.debug("REST request to get Blabla : {}", id);
        Optional<Blabla> blabla = blablaService.findOne(id);
        return ResponseUtil.wrapOrNotFound(blabla);
    }
```

Reply:
@pascalgrimaud  Thanks for pointing the formatting problem. I had same problem and fixed it. I will check what went wrong.

Reply:
@pascalgrimaud I have opened a PR at #9060 to fix this.

Reply:
thx a lot @pmverma !